.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,15 +1,16 @@
 ---
 name: Bug
-about: Describe this issue template's purpose here.
-title: ''
-labels: bug
+about: Create a bug-report to help us address errors in the repo.
+title: '[BUG]'
+labels: 'bug'
```

Comment:
```suggestion
labels: 'bug'
```
I dont think we need this line to change. The labels on the bug template works perfectly fine.